cmd/utils/flags.go

```diff
@@ -1233,9 +1233,7 @@ func SetNodeConfig(ctx *cli.Context, cfg *node.Config) {
 	if ctx.GlobalIsSet(LightKDFFlag.Name) {
 		cfg.UseLightweightKDF = ctx.GlobalBool(LightKDFFlag.Name)
 	}
-	if ctx.GlobalIsSet(USBFlag.Name) {
-		cfg.NoUSB = !ctx.GlobalBool(USBFlag.Name)
-	}
+	cfg.NoUSB = !ctx.GlobalIsSet(USBFlag.Name) || !ctx.GlobalBool(USBFlag.Name)
```

Comment:
I think this is not right and it will break the config file.
What needs to happen is: the default of NoUSB in the struct needs to be made `true`.
Then, if the flag is set, it needs to be changed to `false`.

If someone has `NoUSB = false` in the config file, it should still work.

Reply:
Since it doesn't seem to set default values in the deserializer, the default has to be set either before or after deserializing. How do you propose to distinguish the case when `NoUSB` is absent from the TOML file, from the case when the user explicitly set `NoUSB = false` ?

Reply:
It is very unlikely that people will want to set `NoUSB = false` because until now, `NoUSB = false` was the default and now if they want to enable USB by default they don't have an option for that yet. What I suggest is also add a `USB` command in the config that will default to `false`, and that people can set to `true`.